vendor/arm/mali/mali_profiler.cpp

```diff
@@ -363,6 +363,14 @@ MaliProfiler::MaliProfiler(const GpuCounterSet &enabled_counters) :
 	}
 }
 
+MaliProfiler::~MaliProfiler()
```

Comment:
Can this be removed as it is [implemented here](https://github.com/ARM-software/HWCPipe/pull/34/files#diff-ca3abba14a5fc4c875fcdecf08e1ec0bR58)?

Reply:
Yes, if we'll come up with a conclusion on what do you want to pull-in I can clean up the rest

Reply:
Lets focus this PR on removing the stop interface only